### DISCUSSION TITLE
大佬牛逼，我也想学怎么写脚本

### DISCUSSION THREAD

**Discussion by javencpdd** [hoothin/UserScripts#212] [General]
请问有什么推荐的入门案例吗

**Comment by hoothin** [comment]
问我的话，我可能会推荐你先去学数字逻辑编译原理数据结构。
所以还是直接去联系[打老鼠可爱可爱捏](https://meta.appinn.net/t/topic/17705)比较好

**Comment by javencpdd** [comment]
收到，所以还是从基础看起吗
练习逻辑思维，模仿项目，修改项目，开发项目
谢谢大佬
&nbsp;


佳文
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年11月5日(星期六) 上午9:49
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [hoothin/UserScripts] 大佬牛逼，我也想学怎么写脚本 (Discussion #212)





 
问我的话，我可能会推荐你先去学数字逻辑编译原理数据结构。
 所以还是直接去联系打老鼠可爱可爱捏比较好
